test/fixtures/IL-2015.json

```diff
@@ -0,0 +1,308 @@
+[
```

Comment:
How are the dates calculated?

Fixtures are OK for testing but we need to have a reliable way to calculate dates for any year in the future or in the past.

Reply:
Please check CONTRIBUTING.md